apps/passport-client/components/screens/NoWASMScreen.tsx

```diff
@@ -0,0 +1,20 @@
+import { H2, Spacer, TextCenter } from "../core";
+import { AppContainer } from "../shared/AppContainer";
+
+export function NoWASMScreen() {
+  return (
+    <>
+      <AppContainer bg="primary">
+        <Spacer h={64} />
+        <TextCenter>
+          <H2>No Web Assembly</H2>
+          <Spacer h={24} />
+          Sorry, your device does not currently support Web Assembly. Please
+          consider enabling it, or alternatively, try using Zupass on a
+          different device.
```

Comment:
Only adjustment besides minor wording tweaks is that I would add a link on which browser do support webassembly.

Here's a suggestion:
```suggestion
         Your browser does not currently support WebAssembly, which is
         required for Zupass to generate zero-knowledge proofs. Try using
         Zupass on a different browser, and check that WebAssembly
         is not disabled. You can view a list of supported browsers
         <a href="https://webassembly.org/roadmap/">here</a>.
```

```html
Your browser does not currently support WebAssembly, which is required for Zupass to generate zero-knowledge proofs. Please try using Zupass on a different browser. You can view a list of WebAss
```